lib/utils/age.ts

```diff
@@ -0,0 +1,12 @@
+import { differenceInYears, format, isValid, parseISO } from "date-fns";
+
+export const isUnderEighteen = (dob: string) => {
+  const isValidDate = parseISO(dob);
+  if (!isValid(isValidDate)) {
+    return true;
+  }
+
+  const dobDate = format(new Date(dob), "yyyy-MM-dd");
```

Comment:
I don't think you need to format it here, isValidDate is parsed into a Date object. So we can just compare that in differenceInYears